stats/src/test/java/com/facebook/airlift/stats/TestQuantileDigest.java

```diff
@@ -766,6 +766,62 @@ public void testMergeSameLevel()
         assertEquals(b.getNodeCount(), 1);
     }
 
+    @Test
```

Comment:
Can we add a test where we add value with a very small weight, which when rescaled ends up disappearing?  Per our discussion with Mehrdad :D

Reply:
Another thing: what happens if the digest is empty, but we rescale by infinity?  Will this result in a count of infinity with a tree which is empty (incorrect state)?